### ISSUE TITLE
Support python 3.11

### ISSUE THREAD

**Issue by danielhoherd** [astronomer/telescope#187]
python 3.11 is 6 months old.

https://github.com/astronomer/telescope/blob/main/pyproject.toml#L21

```
$ pip install astronomer-telescope
ERROR: Ignored the following versions that require a different python version: 0.9.1 Requires-Python >=3.6.2,<3.11; 3.0.0 Requires-Python >=3.6.2,<3.11; 3.0.1 Requires-Python >=3.6.2,<3.11; 3.0.2 Requires-Python >=3.6.2,<3.11; 3.1.0 Requires-Python >=3.6.2,<3.11
ERROR: Could not find a version that satisfies the requirement astronomer-telescope (from versions: none)
ERROR: No matching distribution found for astronomer-telescope
```

**Comment by fritz-astronomer** [comment]
Can just bump this line, and run tests 
https://github.com/astronomer/telescope/blob/main/pyproject.toml#L21

**Comment by frankcash** [comment]
(apache-airflow only supports 3.7, 3.9, 3.9 and 3.10 currently)

**Comment by fritz-astronomer** [comment]
closed via telescope 3.2.0
